mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java

```diff
@@ -651,6 +659,8 @@ private void pushTurnstileEvent() {
     event.put(MapboxEvent.KEY_CREATED, TelemetryUtils.generateCreateDate(null));
     event.put(MapboxEvent.KEY_USER_ID, mapboxVendorId);
     event.put(MapboxEvent.KEY_ENABLED_TELEMETRY, isTelemetryEnabled());
+    event.put(MapboxEvent.KEY_SDK_IDENTIFIER, sdkIdentifier);
```

Comment:
👀 https://github.com/mapbox/mapbox-java/pull/574#issuecomment-331575490

> for now `null` values for the legacy Maps SDKs won't break anything.

We should send `null` values when not calling the initialize method that uses `sdkIdentifier` and `sdkVersion` explicitly. 👀 https://github.com/mapbox/mapbox-java/blob/master/mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/navigation/MapboxNavigationEvent.java#L284-L290